lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -52,18 +61,26 @@ def deposit_file(file_name, swift_container)
       # for update: construct hash for key/value pairs as strings,
       # and metadata as additional key/value string pairs in the hash
       headers = { 'etag' => checksum,
-                  'content-type' => 'application/x-tar' }.merge(metadata)
+                  'content-type' => 'application/x-tar',
+                  'X-Auth-New-Token' => 'true' }.merge(metadata)
       deposited_file = era_container.object(file_base_name)
       deposited_file.write(File.open(file_name), headers)
     else
       # for creating new: construct hash with symbols as keys, add metadata as a hash within the header hash
       headers = { etag: checksum,
                   content_type: 'application/x-tar',
-                  metadata: metadata }
+                  metadata: metadata,
+                  'X-Auth-New-Token' => 'true' }
```

Comment:
https://stackoverflow.com/a/16525684 applies to swauth, and I can't find mention of it in https://github.com/openstack/swift/ or https://github.com/ruby-openstack/ruby-openstack.  Do you know what auth we're using with local or OLRC Swift?

Reply:
We're using keystone v1 for local swift and keystone v3 for OLRC